doc/build-windows-mingw.md

```diff
@@ -60,7 +60,7 @@ example /usr/src/bitcoin, AND not under /mnt/d/. If this is not the case the dep
 This means you cannot use a directory that is located directly on the host Windows file system to perform the build.
```

Comment:
Line above this one mentions autoconf. Should that be changed?

Reply:
Our depends still use `autoconf`, don't they?

Reply:
Ah yes, ignore me.